proxy/src/auth/backend/classic.rs

```diff
@@ -53,11 +53,11 @@ pub(super) async fn authenticate(
 
     let mut num_retries = 0;
     let mut node = loop {
-        num_retries += 1;
-        match try_wake(api, extra, creds).await? {
+        info!("compute node's state has likely changed; requesting a wake-up");
```

Comment:
in proy/src/proxy.rs this info message is outside of the loop. is it really needed inside the loop here?